src/common/user.ts

```diff
@@ -87,7 +89,26 @@ export const deleteUser = async (
       await entityManager
         .getRepository(Post)
         .update({ scoutId: userId }, { scoutId: null });
+      await entityManager.getRepository(UserTransaction).update(
```

Comment:
Can we somehow keep the original id? maybe in a flag or anything that can help us track if needed the original user?

Reply:
I was thinking original Ids would be kept on njord side, we can trace through transaction id and idempotencyKey

Reply:
sounds good